src/debug-agent.cc

```diff
@@ -149,6 +154,9 @@ void Agent::Stop() {
   CHECK_EQ(err, 0);
 
   uv_close(reinterpret_cast<uv_handle_t*>(&child_signal_), nullptr);
+  // Close all remaining handles:
+  // uv_loop_close will return UV_EBUSY for handles which are not closed.
+  uv_walk(&child_loop_, close_handle, nullptr);
```

Comment:
This is a workaround. I can't explain why there are always two pipe handles left referenced. 
 
```
[--I] signal   0x30606e8
[-AI] async    0x3060530
[---] async    0x30603e8
[R--] pipe     0x7fd8a0058520
[R--] pipe     0x7fd8a0064260
```